nspanel_blueprint.yaml

```diff
@@ -943,6 +943,28 @@ blueprint:
             default: []
             selector:
               action:
+          left_button_state_entity:
+            name: Left hardware button - State Entity (Optional)
+            description: >
```

Comment:
What about this?

```yaml
  description: >
    Specifies the entity reflecting the state of the left button's indicator bar. 

    If not set, the state of the controlled entity itself will dictate the indicator's state.
```

Reply:
Feel free to give your touch... I just wanna make clear that this is about the indicator bar.